### PR TITLE
Fixing EQUINOX header (hopefully for the last time)

### PR DESCRIPTION
**Pull request by wtgee**
* `Observation.status` will return string, e.g. `J2000`.
* `Observatory.get_standard_headers` will convert to float, e.g. `2000.`
* Always assume a J2000 and don't allow `None` to be set.
* Clean up some comments and make some of the line breaks easier to read.

### CODE REVIEW

**Comment by jamessynge** [comment]
We have a ways to go on coverage. Sigh.

File: pocs/observatory.py [hunk 1]
```diff
@@ -379,6 +380,14 @@ def get_standard_headers(self, observation=None):
         # Add observation metadata
         headers.update(observation.status())
 
+        # Explicitly convert EQUINOX to float
+        try:
+            equinox = float(headers['equinox'].replace('J', ''))
+        except BaseException:
+            equinox = 2000.  # We assume J2000
+        finally:
```
Comment (jamessynge) [inline pocs/observatory.py]: Nit: Doesn't need to be a finally clause.

File: pocs/scheduler/field.py [hunk 2]
```diff
@@ -13,13 +13,20 @@ def __init__(self, name, position, equinox='J2000', **kwargs):
...
 
         Arguments:
-            name {str} -- Name of the field, typically the name of object at center `position`
-            position {str} -- Center of field, can be anything accepted by `~astropy.coordinates.SkyCoord`
-            **kwargs {dict} -- Additional keywords to be passed to `astroplan.ObservingBlock`
+            name {str} -- Name of the field, typically the name of object at
+                center `position`
+            position {str} -- Center of field, can be anything accepted by
+                `~astropy.coordinates.SkyCoord`
+            **kwargs {dict} -- Additional keywords to be passed to
```
Comment (jamessynge) [inline pocs/scheduler/field.py]: Looks like you might be wrapping at under 100.

File: pocs/state/states/default/pointing.py [hunk 3]
```diff
@@ -68,23 +68,24 @@ def on_enter(event_data):
...
+                pocs.say("Ok, I've got the pointing picture, let's see how close we are.")
 
-                pocs.logger.debug("Pointing Coords: {}".format(
-                    pointing_image.pointing))
-                pocs.logger.debug("Pointing Error: {}".format(
-                    pointing_image.pointing_error))
+                pocs.logger.debug("Pointing Coords: {}", pointing_image.pointing)
+                pocs.logger.debug("Pointing Error: {}", pointing_image.pointing_error)
 
         # separation = pointing_image.pointing_error.magnitude.value
```
Comment (jamessynge) [inline pocs/state/states/default/pointing.py]: Commented out code (yes, I'm guilty of the same).

File: pocs/state/states/default/pointing.py [hunk 4]
```diff
@@ -104,5 +105,4 @@ def on_enter(event_data):
         pocs.next_state = 'tracking'
 
     except Exception as e:
-        pocs.say(
-            "Hmm, I had a problem checking the pointing error. Sending to parking. {}".format(e))
+        pocs.say("Hmm, I had a problem checking the pointing error. Sending to parking. {}", e)
```
Comment (jamessynge) [inline pocs/state/states/default/pointing.py]: It feels like "sending to parking" is action at a distance given the next_state = parking way at the top.

Comment (wtgee) [reply pocs/state/states/default/pointing.py]: Sort of yes. All states (should) specify `parking` as the next state at the top of the file and only by proper action does it get changed to a different state. This was aiming to bring consistency.

File: pocs/observatory.py (hunk 1 above)
Comment (wtgee) [reply pocs/observatory.py]: I was mostly trying to make that a contained block. Can remove.

**Pull request merged**